add_a_custom_check.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>We can pass objects for display to the <code>CheckResult</code>. Objects for display should be of type: html string, dataframe or a function that plots a graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>